strapi/strapi
# feat(dts): support models and contentTypes

### What does it do?

- Includes strapi models alongside contentTypes to delete any entries belonging to models included in the DTS import

### Why is it needed?

- e2e tests are currently using DTS to reset the database. History is a new feature using models over contentTypes. The data is not being reset since currently DTS does not take models into consideration.

### How to test it?

- Create a model using the models API
- Export data using DTS
- Make changes to add entries to the model's table
- Import data and see that the model data was reset


The easiest way to test is maybe on this branch: https://github.com/strapi/strapi/pull/19525 where the patch is in place as well. The history-versions model is used in the data import for each test.

co-authored by @Convly
